problem-spring-web-autoconfigure/src/main/java/org/zalando/problem/spring/web/autoconfigure/security/SecurityConfiguration.java

```diff
@@ -14,6 +16,7 @@
 @Configuration
 @ConditionalOnClass(WebSecurityConfigurerAdapter.class) //only when spring-security is in classpath
 @Import(SecurityProblemSupport.class)
+@Order(Ordered.LOWEST_PRECEDENCE - 21)
```

Comment:
That `21` deserves a comment (pretty much just what you wrote in the PR itself).

Reply:
good point. done